dto/src/main/java/org/jboss/pnc/rex/dto/TaskDTO.java

```diff
@@ -60,4 +60,6 @@ public class TaskDTO {
     public Set<String> dependants = new HashSet<>();
 
     public Set<String> dependencies = new HashSet<>();
+
+    public ConfigurationDTO configuration = new ConfigurationDTO();
```

Comment:
I've just remembered. You have to add it also into CreateTaskDTO

Reply:
done